package.json

```diff
@@ -113,7 +114,7 @@
     "linc": "git diff --name-only --diff-filter=ACMRTUB `git merge-base HEAD master` | grep '\\.js$' | xargs eslint --",
     "lint": "node ./scripts/tasks/eslint.js",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
-    "test": "jest",
+    "test": "jest && bundlesize",
```

Comment:
I don't think this will work. `jest` won't perform the build. We have a separate task that's used for CI.
Somewhere [here](https://github.com/facebook/react/blob/f9af9aacd3c14288cd5869ff23cc71cdfb54d447/scripts/circleci/test_entry_point.sh#L27) is probably the right place for it.